src/index.js

```diff
@@ -107,12 +107,20 @@ function ClientApi(options) {
   this.websocket = new Websocket(this);
 }
 
-ClientApi.prototype.requestNew = async (url, data) => {
+ClientApi.prototype.requestNew = async (url, data, format) => {
   try {
     if (data) {
       requestConfig.params = { ...requestConfig.params, ...data };
     }
-    let response = await axios.get(url, requestConfig);
+    let response;
+    if (format === 'other') {
+      response = await axios.get(url, {
+        ...requestConfig,
+        baseURL: BASE_URL_OTHER,
+      });
+    } else {
+      response = await axios.get(url, requestConfig);
```

Comment:
This should be called once after the if statement.

Reply:
actually I am not updating the value in request config variable, rather over riding its value once in the if condition, if we change the value of request config variable, we need to bring it back to default one after the call to the other endpoint is made

Reply:
I'm aware but we should call it only once.

Reply:
So you're suggesting to change the requestConfig value and then revert it once the call is made?

Reply:
Using a new variable rather than reuse the main request config.